frontend/gpay_agent/lib/business_verification_views/business_verification_view.dart

```diff
@@ -1,173 +1,472 @@
+          _buildSilverAppBar(appbarHeight),
+          SliverList(
+            delegate: SliverChildBuilderDelegate(
+                  (BuildContext context, int index) {
+                if (index > 9) return null;
+                switch (index) {
+                  case 0:
+                    return _buildHeaderItem(headerHeight, headerTextSize, iconType.face, headerName.merchant_details);
+                  case 1:
+                    return _buildListItem(itemHeight, itemTextSize, headerName.merchant_details);
+                  case 2:
+                    return _buildHeaderItem(headerHeight, headerTextSize, iconType.store, headerName.business_details);
+                  case 3:
+                    return _buildListItem(itemHeight+20, itemTextSize, headerName.business_details);
+                  case 4:
+                    return _buildHeaderItem(headerHeight, headerTextSize, iconType.phone, headerName.business_phone);
+                  case 5:
+                    return _buildListItem(itemHeight, itemTextSize, headerName.business_phone);
+                  case 6:
+                    return _buildButtonItem(buttonHeight, buttonTextSize, verificationType.finish);
+                  case 7:
+                    return _buildButtonItem(buttonHeight, buttonTextSize, verificationType.revisit);
+                  case 8:
+                    return _buildButtonItem(buttonHeight, buttonTextSize, verificationType.store_does_not_exist);
+                  case 9:
+                    return _buildButtonItem(buttonHeight, buttonTextSize, verificationType.cancel);
+                  default:
+                    return null;
+                }
+              },
+            ),
+          )
+        ],
+      ),
+    );
+  }
+
+  /// SilverAppBar displaying Merchant Name and pins the app bar to screen while scrolling.
+  ///
+  /// Takes height of the appbar as parameter.
+  /// Contains leading back button and pop up menu on the right.
+  SliverAppBar _buildSilverAppBar(double height) {
+    double expandedHeight = height;
+    return SliverAppBar(
+      expandedHeight: expandedHeight,
+      floating: false,
+      pinned: true,
+      snap: false,
+      flexibleSpace: FlexibleSpaceBar(
+        title: Text(
+            globals.merchantName,
+        ),
+        centerTitle: true,
+      ),
+      leading: IconButton(
+        icon: Icon(
+          Icons.arrow_back,
+          color: Colors.white,
+        ),
+        onPressed: () {
+          Navigator.pop(context);
+        },
+      ),
+      actions: <Widget>[_buildPopUpMenu()],
+    );
+  }
+
+  /// Defines action to be taken when pop up menu items are selected.
   void choiceAction(String choice) {
     if (choice == VerificationMenuItems.Cancel) {
-      print('Discard Verification');
       //When ever user clicks on discard verification it opens up verification failed
       //TODO: Need to return to home page after cancelling verification
+      //runApp(MyApp());
```

Comment:
please remove comments that are not required.

Reply:
removed